doc/geany.txt

```diff
@@ -3707,6 +3740,13 @@ Move document right             Ctrl-Shift-PageDown       Changes the current do
 Move document first                                       Moves the current document to the first position.
 
 Move document last                                        Moves the current document to the last position.
+
+Sort tabs based on filename                               Sort tabs based on their filenames or base names.
+
+Sort tabs based on pathname                               Sort tabs based on their full path names.
+                                                          The tabs get sorted similar to how files are
+                                                          arranged in the documents list when Show Paths is
+                                                          enabled.
 =============================== ========================= ==================================================
 
 
```

Comment:
I'm not sure which one is more correct: to say "sort based on filename", or to say "sort by filename".  I find the former more technically correct, even though the latter is more common.